codewiki/src/be/documentation_generator.py

```
@@ -10,87 +10,90 @@
                     metadata["files_generated"].append(file_path)
         except Exception as e:
             logger.warning(f"Could not list generated files: {e}")
-        
+
         metadata_path = os.path.join(working_dir, "metadata.json")
         file_manager.save_json(metadata, metadata_path)
 
-    
-    def get_processing_order(self, module_tree: Dict[str, Any], parent_path: List[str] = []) -> List[tuple[List[str], str]]:
+    def get_processing_order(
+        self, module_tree: Dict[str, Any], parent_path: List[str] = []
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)